upcase_booklist_app/templates/book_list.html

```diff
@@ -0,0 +1,34 @@
+<!doctype html>
```

Comment:
Recommend name of template to be `books_index.html` or `books.html`. Quite conventional.